content/docs/2.0/migration.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "Migration Guide"
++++
+
+## Migrating from KEDA v1 to v2
+
+Please note that you **can not** run both KEDA v1 and v2 on the same Kubernetes cluster. You need to [uninstall](../../1.5/deploy) KEDA v1 first, in order to [install](../deploy) and use KEDA v2.
+
+KEDA v2 is using a new namespace for it's Custom Resources: `keda.sh` instead of `keda.k8s.io` and introduces a new Custom Resource for scaling of Jobs. See full details on KEDA Custom Resources [here](../concepts/#custom-resources-crd).
+
+### Scaling of Deployments
+In order to scale `Deployments` with KEDA v2, you need to do only a few modifications to existing v1 `ScaledObjects` definitions, so they comply with v2:
+- change the value of `apiVersion` property from `keda.k8s.io/v1alpha1` to `keda.sh/v1alpha1`
+- rename property `spec.scaleTargetRef.deploymentName` to `spec.scaleTargetRef.name`
+
+Please see the examples below or refer to the full [v2 ScaledObject Specification](../concepts/scaling-deployments/#scaledobject-spec)
+
+ **Example of v1 ScaledObject**
+```yaml
+apiVersion: keda.k8s.io/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {scaled-object-name}
+spec:
+  scaleTargetRef:
+    deploymentName: {deployment-name} 
+    containerName:  {container-name} 
+  pollingInterval: 30  
+  cooldownPeriod:  300 
+  minReplicaCount: 0   
+  maxReplicaCount: 100 
+  triggers:
+  # {list of triggers to activate the deployment}
+```
+
+**Example of v2 ScaledObject**
+```yaml
+apiVersion: keda.sh/v1alpha1            #  <--- Property value was changed
+kind: ScaledObject
+metadata:
+  name: {scaled-object-name}
+spec:
+  scaleTargetRef:
+    name: {deployment-name}             #  <--- Property name was changed
+    containerName: {container-name}     
+  pollingInterval: 30            
+  cooldownPeriod:  300           
+  minReplicaCount: 0             
+  maxReplicaCount: 100           
+  triggers:
+  # {list of triggers to activate the deployment}
+```
+
+
+### Scaling of Jobs
```

Comment:
If it's a seperate PR, maybe best to leave out for now?

Reply:
I wanted to add the Jobs part stub, so we have the basic structure and don't forget about it :) It should be added anyway. But if it is a big problem, I can remove it

Reply:
It's ok!